pyvista/core/utilities/features.py

```diff
@@ -13,8 +13,11 @@
 from .helpers import wrap
 
 
-def voxelize(mesh, density=None, check_surface=True):
-    """Voxelize mesh to UnstructuredGrid.
+def voxelize(mesh, density=None, vol=False, check_surface=True):
+    """Voxelize mesh to create an UnstructuredGrid or a StructuredGrid.
+       1. Voxelize the input mesh : UnstructuredGrid
+       2. Create a voxel volume of the input mesh. Volume is constructed to 
+          boundaries of input mesh : StructuredGrid
```

Comment:
I suspect our docstrings formatter will complain about this. Might need to separate the list from the first sentence with an empty line